src/providers/fail_over.c

```diff
@@ -46,6 +46,10 @@
 #define DEFAULT_SERVER_STATUS SERVER_NAME_NOT_RESOLVED
 #define DEFAULT_SRV_STATUS SRV_NEUTRAL
 
+#ifndef _PATH_HOSTS
+#define _PATH_HOSTS "/etc/hosts"
```

Comment:
@sumit-bose, maybe it would be better to get rid of `_PATH_HOSTS` usage at all?

It's used only in the one place - to log an error message:
https://github.com/SSSD/sssd/blob/8bfc88e49d9ac9d448c5ecfa80c7629c04cc64d4/src/providers/fail_over.c#L1339

Not sure why it is important to have a different message depending on `resolv_status`, but in any case it can be done without using `_PATH_HOSTS`.

Reply:
Hi,

about the different messages, I think it would be good to know if the lookup failed while trying to get local information or the DNS. But instead of the filename some like "your local hosts file" or similar can be used as well. If filenames are used I prefer defines.

bye,
Sumit

Reply:
> But instead of the filename some like "your local hosts file" or similar can be used

@fossdd, could you please proceed this way and remove `_PATH_HOSTS` usage altogether?

Reply:
Thanks.